.circleci/config.yml

```diff
@@ -245,14 +246,17 @@ jobs:
       - checkout
       - attach_workspace:
           at: ~/wp-calypso
-      - run: *update-node
       - run:
           name: Install Linux deps
           command: |
             sudo apt update
             sudo apt-get install -y libsecret-1-dev
       - restore_cache: *restore-yarn-cache
-      - run: *yarn-install
+      # Since Node and Yarn are both bundled in the image, no need to add nvm and
+      # yarn ourselves. So we run the command directly here.
+      - run:
```

Comment:
Another side benefit is that `nvm use` would keep us on the repo-wide version of 18.13, instead of the bundled version which works for this build (18.10)